Client.php

```diff
@@ -82,9 +81,13 @@ class Client extends Component
     public $contentLoggingMaxSize = 2000;
 
     /**
-     * @var Transport|array|string|callable HTTP message transport.
+     * @var Transport|array|string|callable HTTP message transport instance.
+     * Note that the type of this property differs in getter and setter.
+     *
+     * @see getTransport()
+     * @see setTransport()
      */
-    private $_transport = 'yii\httpclient\StreamTransport';
+    private $transport = 'yii\httpclient\StreamTransport';
```

Comment:
@klimov-paul I've changed the signatures of `$_variable` to  `$variable` through all code (not only in that class). Can it cause any side effects?

Reply:
Yes - it is a framework convention to prevent conflicts between real properties and virtual attributes provided by getters/setters. PSR-2 still allows such names when there is valid reason for it.
Also PSR-2 does not have hard limit for line length, so 👎  for unnecessary docs wrapping.

Reply:
@rob006 OK, I'll rewrite PR.

Reply:
Also at https://github.com/yiisoft/yii2/tree/master/cs there is a working, but still unfinished CS validation based on php-cs-fixer. It may be easier for you to play with php-cs-fixer and existing config than reinventing the same thing with CodeSniffer.